src/types/Contact.ts

```diff
@@ -108,7 +111,23 @@ export interface Contact {
         view: string;
     };
 
-    social: { type: Contact.SocialNetwork; username: string }[];
+    enrichments: {
+        languages: Language['code'][];
+        medium_types: Contact.MediumType[];
+        urls: string[];
+        phone_numbers: Contact.PhoneNumber[];
+        social: { type: Contact.SocialNetwork; username: string }[];
+        periodicity: Contact.Periodicity | null;
```

Comment:
Did you change your mind about always using arrays?

Reply:
Yes. After thinking more on it, I've decided that repeating the structure of the contact API object is more beneficial.